src/WSManProcessor.ts

```diff
@@ -190,14 +190,14 @@ export class WSManProcessor {
     try {
       const amtstack = this.getAmtStack(clientId, amtuser, amtpass)
       await amtstack.BatchEnum('', [action], (stack, name, jsonResponse, status) => {
-        if (status != 200) {
+        if (status !== 200) {
           console.log('Request failed during hardware_info BatchEnum Exec.')
         } else {
           this.logger.info(`batchEnum request succeeded for clientId: ${clientId}, action:${action}.`)
         }
         clientObj.payload = jsonResponse
       })
-      if (clientObj.socketConn?.onStateChange && clientObj.readyState == undefined) {
```

Comment:
was this the change we talked about earlier?? that is might change the functionality

Reply:
a few more instances below

Reply:
this wont, null is the same as undefined with ==

Reply:
ok thanks for the clarification